site/content/docs/5.2/utilities/shadows.md

```diff
@@ -11,10 +11,10 @@ toc: true
 While shadows on components are disabled by default in Bootstrap and can be enabled via `$enable-shadows`, you can also quickly add or remove a shadow with our `box-shadow` utility classes. Includes support for `.shadow-none` and three default sizes (which have associated variables to match).
 
 {{< example >}}
-<div class="shadow-none p-3 mb-5 bg-light rounded">No shadow</div>
-<div class="shadow-sm p-3 mb-5 bg-body rounded">Small shadow</div>
-<div class="shadow p-3 mb-5 bg-body rounded">Regular shadow</div>
-<div class="shadow-lg p-3 mb-5 bg-body rounded">Larger shadow</div>
+<div class="shadow-none p-3 mb-5 bg-body-tertiary rounded">No shadow</div>
```

Comment:
Not linked to this modification especially but I saw the following in this page. In dark mode, we don't see the box shadows:

![Screenshot 2022-11-17 at 09 10 02](https://user-images.githubusercontent.com/17381666/202391448-cd4e2baa-bb98-4a31-ad79-3623f4a36215.png)

This is probably due to the fact that `$box-shadow*` are defined based on `$black` which stays black in dark mode.

I haven't checked in details but we could do something like the following to have a dynamic utility:

```scss
$box-shadow:                  0 .5rem 1rem rgba(var(--#{$prefix}body-color-rgb), .15) !default;
$box-shadow-sm:               0 .125rem .25rem rgba(var(--#{$prefix}body-color-rgb), .075) !default;
$box-shadow-lg:               0 1rem 3rem rgba(var(--#{$prefix}body-color-rgb), .175) !default;
$box-shadow-inset:            inset 0 1px 2px rgba(var(--#{$prefix}body-color-rgb), .075) !default;
```